preprocessor/src/main/java/com/azure/autorest/preprocessor/namer/CodeNamer.java

```diff
@@ -226,6 +240,17 @@ protected static String getEscapedReservedName(String name, String appendValue)
         return name;
     }
 
+    protected static String getEscapedReservedNameAndClasses(String name, String appendValue) {
```

Comment:
I put a new RESERVED_WORDS_CLASSES set for types.

There class/annotation names are fine for parameter/proerty.